frontend/src/scenes/sessions/sessionsPlayLogic.ts

```diff
@@ -147,5 +149,44 @@ export const sessionsPlayLogic = kea<sessionsPlayLogicType<SessionPlayerData, Ev
             (index: number, ids: SessionRecordingId[], pagination: Record<string, any> | null) =>
                 index > -1 && (index < ids.length - 1 || pagination !== null),
         ],
+        session: [
+            (selectors) => [selectors.sessionRecordingId, selectors.sessions],
+            (id: SessionRecordingId, sessions: Array<SessionType>): SessionType | null => {
+                const [session] = sessions.filter((s) => s.session_recording_ids.includes(id))
+                return session
+            },
+        ],
+        highlightedSessionEvents: [
+            (selectors) => [selectors.session, selectors.loadedSessionEvents],
+            (session, sessionEvents) => {
+                if (!session) {
+                    return []
+                }
+                const events = session.events || sessionEvents[session.global_session_id] || []
+                return events.filter((e) => (session.action_filter_times || []).includes(e.timestamp))
+            },
+        ],
+        shownPlayerEvents: [
+            (selectors) => [selectors.sessionPlayerData, selectors.eventIndex, selectors.highlightedSessionEvents],
+            (sessionPlayerData, eventIndex, events) => {
+                if (!sessionPlayerData) {
+                    return []
+                }
+                const startTime = +moment(sessionPlayerData.start_time)
+
+                const pageChangeEvents = eventIndex.pageChangeEvents().map(({ playerTime, href }) => ({
+                    playerTime,
+                    text: href,
+                    color: 'blue',
```

Comment:
Non-critical. Where are these colors set? They're not in our palette, and don't look great.

Reply:
These are defaults from ant - https://ant.design/components/tag/

Reply:
That makes sense, I'll just add a note to fix this in a subsequent PR. I do need to override ant globals.